qemu/tests/cfg/block_resize.cfg

```diff
@@ -59,3 +59,14 @@
         disk_change_ratio = 1.5 0.5
     RHEL.5:
         need_reboot = yes
+    variants:
+        - @default:
+        - data_plane:
```

Comment:
block_resize is a basic test, make sure it doesn't effect other test when add variants

Reply:
It just effects internal test loop "virtio-win-acceptance.cfg" and after discussion, it's acceptable.